src/Sarif/Autogenerated/SarifRewritingVisitor.cs

```diff
@@ -831,6 +846,14 @@ public virtual SarifLog VisitSarifLog(SarifLog node)
                         node.Runs[index_0] = VisitNullChecked(node.Runs[index_0]);
                     }
                 }
+
+                if (node.InlineExternalPropertyFiles != null)
+                {
+                    for (int index_0 = 0; index_0 < node.InlineExternalPropertyFiles.Count; ++index_0)
+                    {
+                        node.InlineExternalPropertyFiles[index_0] = VisitNullChecked(node.InlineExternalPropertyFiles[index_0]);
+                    }
+                }
```

Comment:
same comment as above: this is strictly not part of this PR, but when i regenerated this file i noticed this additional check and thought of bringing is to attention in case it was mistakenly not included earlier?